spec/features/admin/admin_users_spec.rb

```diff
@@ -93,14 +95,14 @@
       end
 
       it "should show page with  new data" do
-        page.should have_content("bigbang@mail.com")
-        page.should have_content("Big Bang")
+        expect(page).to have_content("bigbang@mail.com")
+        expect(page).to have_content("Big Bang")
       end
 
       it "should change user entry" do
         @simple_user.reload
-        @simple_user.name.should == "Big Bang"
-        @simple_user.is_admin?.should be_true
+        expect(@simple_user.name).to eq("Big Bang")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.